homeassistant/components/asuswrt/const.py

```diff
@@ -20,12 +22,32 @@
 MODE_AP = "ap"
 MODE_ROUTER = "router"
 
+PROTOCOL_HTTP = "http"
+PROTOCOL_HTTPS = "https"
 PROTOCOL_SSH = "ssh"
 PROTOCOL_TELNET = "telnet"
 
 # Sensors
 SENSORS_BYTES = ["sensor_rx_bytes", "sensor_tx_bytes"]
 SENSORS_CONNECTED_DEVICE = ["sensor_connected_device"]
+SENSORS_CPU = [
+    "cpu_total_usage",
+    "cpu1_usage",
+    "cpu2_usage",
+    "cpu3_usage",
+    "cpu4_usage",
+    "cpu5_usage",
+    "cpu6_usage",
+    "cpu7_usage",
+    "cpu8_usage",
+]
 SENSORS_LOAD_AVG = ["sensor_load_avg1", "sensor_load_avg5", "sensor_load_avg15"]
+SENSORS_MEMORY = [
+    "sensor_memory_perc",
+    "sensor_memory_total",
```

Comment:
This value will not change over time, so we should not implement such a sensor.

Reply:
You mean `sensor_memory_total`? It should be fully removed or just represented as sensor without `state_class`?

Reply:
Yes, I mean memory total. If I remember correctly it is not allowed to create entities with static values.

Reply:
Yes, make sense. Removed.